app/metrics.yaml

```diff
@@ -254,3 +260,249 @@ control:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-11-01"
+
+pages:
+  page_load:
+    type: timing_distribution
+    description: >
+      Measuring how long each page takes to load.
+    time_unit: nanosecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+
+immersive:
+  duration:
+    type: timing_distribution
+    description: >
+      Measuring how long users stay in the immersive mode.
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+
+windows:
+  duration:
+    type: timing_distribution
+    description: >
+      Measuring how long windows keep opening.
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  movement:
+    type: counter
+    description: >
+      Counting how many times of moving a window in a session.
+    send_in_pings:
+      - session-end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  resize:
+    type: counter
+    description: >
+      Counting how many times of resizing a window in a session.
+    send_in_pings:
+      - session-end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  active_in_left_time:
+    type: timing_distribution
+    description: >
+      Measuring how long a window is placed at the left in a session.
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
```

Comment:
What do you expect the maximum value of this to be? If it's measured in days, you want to use `millisecond` here (and the others below):

Reply:
This measurement is collected when a session is ended. That happens when our app jumps to the background. It usually happens by hours don't need days. IIUC, `microsecond` range is from `1μs <= x <= ~6.94 days`, so it should be long enough for us to measure at least one day. This also reminds me I should make windows.duration to be `millisecond`. Our main window will keep alive until our app is killed, it should be able to last for a couple of days.

Reply:
Perfect!